### PR TITLE
[UPDATED] the home page svg file 

### LINKED ISSUE #5720: Change homepage diagram
**Describe the change you'd like to see**
The current homepage diagram has an inaccessible color scheme. We should update the website to have more cohesive and higher contrast colors.

**Additional context**
cc @mmejia02 to add the svg file for new diagram

### PR DESCRIPTION
**Pull request by vinfinity7**
Fixes #5720 

The Home page svg file has been changed to new one provided to me 👍

### CODE REVIEW

**Review by Cali0707 — CHANGES_REQUESTED** [review]
@vinfinity7 you will have to replace the existing home image for the flowchart for this change to work, rather than create a new one. If you look at the site preview for this PR (https://deploy-preview-5721--knative.netlify.app/docs/), it is still showing the old image.

**Comment by vinfinity7** [comment]
@Cali0707 please review the changes

**Review by Cali0707 — CHANGES_REQUESTED** [review]
@vinfinity7 the file `overrides/images/home-images/knative_flowchart_graphic.svg` is still here, even though it looks like you've deleted it's contents. Can you delete it fully?

Also, you have been adding a new file, when what this issue is asking you to do is to _replace_ the existing file. Can you find the file that is currently being used in the homepage and replace it, rather than adding any new files and/or changing the `src` of the `img` tag?

**Comment by vinfinity7** [comment]
So i've commited the delete of the original file @Cali0707 , kindly review the deploy and replace the src

**Comment by vinfinity7** [comment]
@Cali0707 you might wanna lead me on this !

**Comment by Cali0707** [comment]
@vinfinity7 if you look here: https://github.com/knative/docs/tree/main/docs/images/home-images you will see that there already is a file with the old image. What you should do is _replace_ that file with the new one, rather than creating a new file. Thanks to git, we will always be able to go back and find the old file if needed.

**Comment by vinfinity7** [comment]
image
 _**Replaced**_   all three instances of the image 👍 @Cali0707

File: overrides/images/home-images/knative_flowchart_graphic.svg
Comment (Cali0707) [inline overrides/images/home-images/knative_flowchart_graphic.svg:1]: @vinfinity7 you're still adding this file. Can you delete it?

Comment (vinfinity7) [reply overrides/images/home-images/knative_flowchart_graphic.svg:1]: @Cali0707 You can look into this now !

File: overrides/home.html [hunk 1]
```diff
@@ -155,7 +155,7 @@ <h1>Knative Components</h1>
         Eventing work together to automate and manage tasks and applications.
       </p>
       <div class="components-content">
-        <img class="components-img" src="images/home-images/knative_flowchart_graphic.svg" alt="" draggable="false" />
+        <img class="components-img" src="../docs/images/home-images/knative_flowchart_graphic.svg" alt="" draggable="false" />
```
Comment (Cali0707) [inline overrides/home.html:158]:
```suggestion
        <img class="components-img" src="images/home-images/knative_flowchart_graphic.svg" alt="" draggable="false" />
```
There shouldn't be any need to change this, can you revert this change and we can see if the home page builds properly?

Comment (vinfinity7) [reply overrides/home.html:158]: yes , but this also works right , deploy preview is on point @Cali0707

Comment (Cali0707) [reply overrides/home.html:158]: yes, but I would rather not make unnecessary changes -  can you revert this change since it shouldn't be needed?

Comment (vinfinity7) [reply overrides/home.html]: it is fine now @Cali0707

**Comment by vinfinity7** [comment]
@Cali0707 why is the Knative Verify Build test failing ?

**Comment by Cali0707** [comment]
> @Cali0707 why is the Knative Verify Build test failing ?

This is an unrelated issue to your changes, I wouldn't worry about it
